fs/fs.go

```diff
@@ -295,11 +296,38 @@ func (i *RealFsInfo) addDockerImagesLabel(context Context, mounts []*mount.Info)
 }
 
 func (i *RealFsInfo) addCrioImagesLabel(context Context, mounts []*mount.Info) {
+	labelCrioImageOrContainers := LabelCrioContainers
+	// If imagestore is not specified, let's fall back to the original case.
+	// Everything will be stored in crio-images
+	if context.Crio.ImageStore == "" {
+		labelCrioImageOrContainers = LabelCrioImages
+	}
 	if context.Crio.Root != "" {
 		crioPath := context.Crio.Root
 		crioImagePaths := map[string]struct{}{
 			"/": {},
 		}
+		for _, dir := range []string{"devicemapper", "btrfs", "aufs", "overlay", "zfs"} {
```

Comment:
why not use StorageDriver as well instead of iterating through all of them?

Reply:
Can you link or explain more on this?

Reply:
the [storage driver](https://github.com/google/cadvisor/pull/3395/files#diff-f427f5cea39f949a5e7e26217e7d444dbafef6923fbd4d10e2012b8ffecf00b2R45) field will be one of these values, and will also be the dir that's being used. instead of iterating through the options, I think you could just use that value

Reply:
I see!

Reply:
Pushed a change for this.